### PR TITLE
PD2-2127: added tests and missing user action view

### PR DESCRIPTION
**Pull request by alexp374**
https://mindojo.atlassian.net/browse/PD2-2127

related https://github.com/mindojo/live-visitors-interview/pull/3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,33 +21,24 @@
...
+    "@vue/test-utils": "^1.0.3",
     "babel-eslint": "^10.1.0",
     "eslint": "^6.7.2",
     "eslint-config-airbnb-base": "14.2.0",
     "eslint-plugin-import": "2.22.0",
     "eslint-plugin-vue": "^6.2.2",
+    "flush-promises": "1.0.2",
     "sass": "1.26.10",
     "sass-loader": "9.0.3",
+    "sinon": "9.0.3",
```
Comment (kundojo) [inline package.json:35]: why sinon if you already have jest? :)

Comment (alexp374) [reply package.json:35]: I think we can use jest, but since we regularly use sinon I kinda used to it

File: src/admin/utils/parseGetParams.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export default function (params) {
+  return params.split('?')
+    .join('').split('&')
+    .reduce((result, x) => {
+      const item = x.split('=');
```
Comment (kundojo) [inline src/admin/utils/parseGetParams.js:5]: `const [key, value] = x.split` will do the work

**Review by kundojo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (kundojo) [reply package.json:35]: but kind of odd use both :)

**Pull request merged**